### PR TITLE
Refactor: Add boundary installer download link

### PR DESCRIPTION
**Pull request by heatlikeheatwave**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-heat-add-boundary-installer-dow-b0c4b7-hashicorp.vercel.app/boundary/install)
- [Asana task](https://app.asana.com/0/1202801212949828/1208629757766424/f)

## 🗒️ What

- Rather than having two components (boundary install and boundary desktop client), I refactored this to only use one component, with the addition of a `className` prop to add the padding between cards. 

## 🤷 Why

- To keep the code DRY

### CODE REVIEW

File: src/views/product-downloads-view/boundary/components/install-callout/index.tsx [hunk 1]
```diff
@@ -10,33 +10,36 @@ import MobileDownloadStandaloneLink from 'components/mobile-download-standalone-
...
 
 /**
  * Render a callout to download the Boundary Desktop Client.
  */
-function DesktopClientCallout({
-	desktopClientProps,
+function InstallCallout({
+	customInstallProps,
 	headingData,
+	className,
```
Comment (johanbrandhorst) [inline src/views/product-downloads-view/boundary/components/install-callout/index.tsx:30]: maybe call this `cardClassName` to make it clear that the class name applies to the card?

File: src/views/product-downloads-view/boundary/components/install-callout/install-callout.module.css [hunk 2]
```diff
@@ -32,3 +28,7 @@
 		grid-template-columns: 16px 1fr 90px;
 	}
 }
+
+.lastCard {
+	margin-top: 32px;
+}
```
Comment (johanbrandhorst) [inline src/views/product-downloads-view/boundary/components/install-callout/install-callout.module.css:34]: Looks like this was moved to `src/views/product-downloads-view/boundary/download.module.css`, is it still needed here?

**Review by johanbrandhorst — APPROVED** [review]
LGTM! Just a few questions. Thanks for the PR to fix my code 😁

File: src/views/product-downloads-view/boundary/components/install-callout/install-callout.module.css (hunk 2 above)
Comment (heatlikeheatwave) [reply src/views/product-downloads-view/boundary/components/install-callout/install-callout.module.css:34]: No, good catch!

**Pull request merged**